static/src/stylesheets/amp/_d2-comments.scss

```diff
@@ -0,0 +1,50 @@
+/* Comments
+   ========================================================================== */
+
+.d2-comment-embedded {
+    background-color: #ffffff;
+    font-family: sans-serif;
+    overflow: hidden;
+    position: relative;
+    margin-bottom: 1.5em;
+    padding-top: 15px;
```

Comment:
The padding and margin are not part of our grid system, any reason for using these magic numbers instead of something from https://github.com/guardian/guss-grid-system